<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-C582COR" failures="2" tests="5" name="firefoxTest" time="41.435" errors="0" timestamp="2022-09-23T20:36:46 IST">
  <testcase classname="TestPackge.TestNGClass2" name="@BeforeMethod facebookloginpage" time="0.001">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.openqa.selenium.WebDriver.get(String)&amp;quot; because &amp;quot;this.driver&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.get(String)" because "this.driver" is null
at TestPackge.TestNGClass2.facebookloginpage(TestNGClass2.java:79)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod facebookloginpage -->
  <testcase classname="TestPackge.TestNGClass2" name="@AfterMethod aftermethod" time="-1.663945587232E9">
    <skipped/>
  </testcase> <!-- @AfterMethod aftermethod -->
  <testcase classname="TestPackge.TestNGClass2" name="@BeforeMethod facebookloginpage" time="-1.663945587235E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod facebookloginpage -->
  <testcase classname="TestPackge.TestNGClass2" name="@AfterMethod aftermethod" time="-1.663945587241E9">
    <skipped/>
  </testcase> <!-- @AfterMethod aftermethod -->
  <testcase classname="TestPackge.TestNGClass2" name="@BeforeMethod facebookloginpage" time="-1.663945587243E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod facebookloginpage -->
  <testcase classname="TestPackge.TestNGClass2" name="@AfterMethod aftermethod" time="-1.663945587246E9">
    <skipped/>
  </testcase> <!-- @AfterMethod aftermethod -->
  <testcase classname="TestPackge.TestNGClass2" name="@AfterClass removePOMClssObjects" time="-1.663945587249E9">
    <skipped/>
  </testcase> <!-- @AfterClass removePOMClssObjects -->
  <testcase classname="TestPackge.TestNGClass2" name="@AfterTest closebrowser" time="-1.663945606129E9">
    <skipped/>
  </testcase> <!-- @AfterTest closebrowser -->
  <testcase classname="TestPackge.TestNGclass" name="@AfterTest closeBrowser" time="-1.663945606131E9">
    <skipped/>
  </testcase> <!-- @AfterTest closeBrowser -->
  <testcase classname="TestPackge.TestNGClass2" name="createnewaccountdisplay" time="0.0">
    <skipped/>
  </testcase> <!-- createnewaccountdisplay -->
  <testcase classname="TestPackge.TestNGClass2" name="loginpagedisplay" time="0.0">
    <skipped/>
  </testcase> <!-- loginpagedisplay -->
  <testcase classname="TestPackge.TestNGClass2" name="featurepagedisplay" time="0.0">
    <skipped/>
  </testcase> <!-- featurepagedisplay -->
  <testcase classname="TestPackge.TestNGclass" name="opencontacttohelpcenterpage" time="1.396">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	url of contact to help center  is not working expected [https://www.messenger./] but found [https://www.messenger.com/help],
	title of page is wrong expected [Messenger Centre] but found [Messenger Help Centre]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
url of contact to help center  is not working expected [https://www.messenger./] but found [https://www.messenger.com/help],
title of page is wrong expected [Messenger Centre] but found [Messenger Help Centre]
at TestPackge.TestNGclass.opencontacttohelpcenterpage(TestNGclass.java:123)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- opencontacttohelpcenterpage -->
  <testcase classname="TestPackge.TestNGclass" name="openmessgenerpage" time="2.676">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	url of openmessnger is not working expected [https://www.messenge] but found [https://www.messenger.com/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
url of openmessnger is not working expected [https://www.messenge] but found [https://www.messenger.com/]
at TestPackge.TestNGclass.openmessgenerpage(TestNGclass.java:101)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- openmessgenerpage -->
</testsuite> <!-- firefoxTest -->
